tests/Unit/Datagrid/PagerTest.php

```diff
@@ -16,14 +16,15 @@
 use Doctrine\ODM\PHPCR\Query\Query as PHPCRQuery;
 use PHPUnit\Framework\TestCase;
 use Sonata\DoctrinePHPCRAdminBundle\Datagrid\Pager;
+use Sonata\DoctrinePHPCRAdminBundle\Datagrid\ProxyQuery;
 
 class PagerTest extends TestCase
 {
     public function setUp()
     {
         $this->pager = new Pager(10);
 
-        $this->proxyQuery = $this->getMockBuilder('Sonata\DoctrinePHPCRAdminBundle\Datagrid\ProxyQuery')
+        $this->proxyQuery = $this->getMockBuilder(ProxyQuery::class)
```

Comment:
please use `createMock()` here instead

Reply:
@ElectricMaxxx did not introduce `getMockBuilder`, maybe this could be done in a separate PR?

Reply:
It could definitely, but he is modifiying this line anyway. No KO criteria, I would do it afterwards, too if he don't want to do it 👍